snapshots/cardano-1.25.1-GHC-8.10.4.yaml

```diff
@@ -108,12 +108,14 @@ packages:
   - ouroboros-network-framework
   - typed-protocols
   - typed-protocols-examples
-  - Win32-network
   # Extra packages not used by cardano-node
   - cardano-client
   - ntp-client
   - ouroboros-consensus-mock
 
+- git: https://github.com/input-output-hk/Win32-network
+  commit: f8a1bcd08608d78e200f600b5e9d2cd9dec233f2
```

Comment:
This commit is different to the one in the other file. I think f71be82 is the correct rev.

Reply:
Good spot.  Thanks!